packages/drivers/odsp-driver/src/odspDocumentDeltaConnection.ts

```diff
@@ -213,6 +214,7 @@ export class OdspDocumentDeltaConnection extends DocumentDeltaConnection impleme
             versions: protocolVersions,
             nonce: uuid(),
         };
+        (connectMessage as any).epoch = epoch;
```

Comment:
I think I would be adding it in interface at server side before submitting this PR. @vladsud